Jenkinsfile

```diff
@@ -60,11 +60,11 @@ pipeline {
         stage("Build & Push Docker Image") {
             steps {
                 script {
-                    docker.withRegistry('',DOCKER_PASS) {
+                    docker.withRegistry('https://index.docker.io/v1/', ${DOCKER_PASS} ) {
```

Comment:
make sure you called the right credentials on that variable, and verify your syntax